src/lua/log.lua

```diff
@@ -395,6 +396,63 @@ local function log_pid()
     return tonumber(ffi.C.log_pid)
 end
 
+local Ratelimit = {
+    interval = 60,
+    burst = 10,
+    emitted = 0,
+    suppressed = 0,
+    start = 0,
+}
```

Comment:
We can use lua RateLimit later for other purposes. It's better to pass interval and burst through `new` method.

Reply:
Actually, these are the default values: you can pass a table to the new method overriding them, e.g.:

```lua
local rl = Ratelimit:new{burst = 15}
assert(rl.burst == 15)
```